lib/gitlib.sh

```diff
@@ -134,7 +134,7 @@ gitlib::github_api_token () {
 # returns 1 on failure
 # Disable shellcheck for dynamically defined variable
 # shellcheck disable=SC2154
-export PROGSTEP[gitlib::github_acls]="CHECK GITHUB AUTH"
```

Comment:
I think shellcheck is complaining about this. May just need to disable it on this line